deployment/modules/process-utils.psm1

```diff
@@ -0,0 +1,76 @@
+# ------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All rights reserved.
+#  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+# ------------------------------------------------------------
+
+# Function to start a process in a new minimized window
+
+Function Start-ProcessInNewTerminalPW {
+    param (
+        [string]$ProcessArgs,
+        [string]$WindowTitle
+    )
+
+    if ($IsLinux) {
+        # Linux-specific code
+        $screenTitle = "-t '$WindowTitle'"
+        $command = "gnome-terminal $screenTitle -- bash -c '$ProcessArgs; exec bash'"
+        if($env:AZUREPS_HOST_ENVIRONMENT)
```

Comment:
Should we throw error/message if it's not Azure Cloud Shell env? I think we check this in calling scripts but not here?

Reply:
I will move the check to the function because first I thought the function did not need to know about this but it does depend on the environment so this can be optimized.

Reply:
Added log of error if wrong environment